token/program-2022/src/processor.rs

```diff
@@ -7524,6 +7533,32 @@ mod tests {
             ),
             Err(ProgramError::IncorrectProgramId)
         );
+
+        // Invalid Extension Type for mint and uninitialized account
+        assert_eq!(
+            do_process_instruction(
+                get_account_data_size(
+                    &program_id,
+                    &mint_key,
+                    &[ExtensionType::Uninitialized]
+                )
+                .unwrap(),
+                vec![&mut invalid_mint_account],
+            ),
+            Err(TokenError::ExtensionTypeMismatch.into())
+        );
+        assert_eq!(
+            do_process_instruction(
+                get_account_data_size(
+                    &program_id,
+                    &mint_key,
+                    &[ExtensionType::MintCloseAuthority]
```

Comment:
to be totally sure, can you add an account extension type here to make sure that the error is still getting triggered?